h/zowetypes.h

```diff
@@ -249,6 +280,9 @@ typedef uint64_t uint64;
 #define INT64_INT(x) ((int)x)
 #define INT_INT64(x) ((int64)x)
 #define CHARPTR_INT64(x) ((int64)x)
+#define INT2PTR(x) ((void*)((int64_t)(x)))
```

Comment:
Do we care about sign extension leading to an incorrect address? I can only think of some 16-bit address like 0xF004 which is cast to 0xFFFFFFFFFFFFF004. 31-bit addresses should not be in danger I guess.

Reply:
A short (16 bit) might have this issue, not an int.

Reply:
Ok, so as long as we use this for `int32_t` only it will work correctly.